test/functionalTest/cases/1316_create_subscription/post_subscriptions.test

```diff
@@ -228,8 +228,8 @@ Date: REGEX(.*)
     "id": "REGEX([0-9a-f]{24})",
     "notification": {
         "attrs": [
-            "humidity",
-            "temperature"
+            "temperature",
```

Comment:
I understand that you have needed to change attrs order in order to "align" with the orther in which they appear in notifications due to that in this case accumulator doesn't run in pretty-print mode?

(Just to check if I'm understanding correctly the change)